contract/views/contract.xml

```diff
@@ -341,7 +360,9 @@
                 <attribute name="string">Customer</attribute>
                 <attribute
                     name="context"
-                >{'default_customer_rank': 1, 'default_supplier_rank': 0, 'res_partner_search_mode': 'customer', 'show_vat': True}</attribute>
+                >{'default_customer_rank': 1, 'default_supplier_rank': 0,
+                    'res_partner_search_mode': 'customer', 'show_vat': True}
+                </attribute>
```

Comment:
Even if it is correct here, I would prefer using `base_view_inheritance_extension` for better extension mechanisms (if some context keys are added at the same place in another addon for eg): https://github.com/OCA/server-tools/tree/18.0/base_view_inheritance_extension